### ISSUE TITLE
clean up JohnTravoltageKeyboardHelpContent

### ISSUE THREAD

**Issue by pixelzoom** [phetsims/john-travoltage#279]
From https://github.com/phetsims/joist/issues/488#issuecomment-384403141:

> I fixed all of the above except for `JohnTravoltageKeyboardHelpContent`.  It's doing something evil at the end of its constructor:

```js
    // the content should be centered in the dialog relative to the description text
    var oldCenter = contentHBox.centerX;
    var newCenter = descriptionVBox.centerX;
    var spacerWidth = newCenter - oldCenter + 2 * DIALOG_MARGIN;
    contentHBox.addChild( new Spacer( spacerWidth, 0 ) );
```

@zepumph from https://github.com/phetsims/joist/issues/488#issuecomment-384798464:

> I removed the panel from JT. I decided to keep the code smell illustrated in https://github.com/phetsims/joist/issues/488#issuecomment-384403141. JT was the first keyboard navigation dialog, and the content is by no means idiomatic. When we publish this sim again, we will update the keyboard dialog content to a consistent look with `HelpContent.js`. I'm fine letting it slide until then. JT is not high priority right now, and would design time for the updated dialog.

**Comment by jessegreenberg** [comment]
> we will update the keyboard dialog content to a consistent look with HelpContent.js

This was done as part of #263, and the Spacer shenanigans have been removed. Closing.
